src/main/groovy/com/bmuschko/gradle/docker/response/container/InspectExecContainerResponseHandler.groovy

```diff
@@ -0,0 +1,21 @@
+package com.bmuschko.gradle.docker.response.container
+
+import org.gradle.api.logging.Logger
+import org.gradle.api.logging.Logging
+
+/**
+ * Logs inspect exec container result.
+ *
+ * @see com.bmuschko.gradle.docker.tasks.container.DockerInspectExecContainer
+ */
+class InspectExecContainerResponseHandler {
```

Comment:
This is just a fallback which prints result to stdout (via logger) when `onNext` is not defined. IMO it is consistent with current implementation of other tasks.